ethereum/verkletrie/build.gradle

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+apply plugin: 'java-library'
+
+jar {
+  archiveBaseName = 'besu-verkletrie'
+  manifest {
+    attributes(
+      'Specification-Title': archiveBaseName,
+      'Specification-Version': project.version,
+      'Implementation-Title': archiveBaseName,
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+dependencies {
+  implementation project(':crypto')
+  implementation project(':ethereum:rlp')
+  implementation project(':ethereum:trie')
+
+  implementation "org.immutables:value-annotations"
```

Comment:
I'm not sure we need all these dependencies but I could be wrong

Reply:
straight copy from merkletrie. The code quickly builds to those with the next PR.